pkg/stub/handler.go

```diff
@@ -333,13 +342,17 @@ func (h *Handler) processImageStreamWatchEvent(is *imagev1.ImageStream, deleted
 	}
 	err = h.upsertImageStream(imagestream, is, srcfg)
 	if err != nil {
+		if kerrors.IsAlreadyExists(err) {
```

Comment:
why would an upsert get an already exists error?  shouldn't it do an update if it already exists?

Reply:
concurrent activity can occur ... delete events for imagestreams that have been previously deleted as part of a samplesresource delete, but then then the new samplesresource creating the new item